test/units/TEST-13-NSPAWN.nspawn.sh

```diff
@@ -1446,4 +1446,28 @@ testcase_unpriv_dir() {
     rm -rf "$root"
 }
 
+testcase_link_journa_hostl() {
+    local root hoge i
+
+    root="$(mktemp -d /var/lib/machines/TEST-13-NSPAWN.link-journal.XXX)"
```

Comment:
/tmp or /var/tmp to avoid the need for a cleanup?

Reply:
/var/lib/machines is a bind mount of a directory under /var/tmp. Hence explicitly using /var/tmp is not necessary. We already use there in many testcases.